pages/builds/parallelizing_builds.md.erb

```diff
@@ -36,11 +36,11 @@ To run the same step over all 5 of the agents we can set the parallelism field f
 
```

Comment:
Is there an example of a time where running the same step 5 times in parallel would be super useful?